### PR TITLE
DO NOT MERGE: adding the APEX records for digitalgov.gov

### PR DESCRIPTION
**Pull request by jeremyzilar**
We are documenting this change here: https://github.com/GSA/digitalgov.gov/issues/100

PRs affecting cloud.gov.tf or a Federalist site must receive approval from a member of the relevant team.

### CODE REVIEW

**Comment by wslack** [comment]
This seems fine but as Eric said, if you are delegating the whole nameserver then you need to replicate the other DNS settings into this file first.

**Comment by wslack** [comment]
Hey @jeremyzilar, because I only own the Federalist side of things, I wanted to make sure you had support. It sounds like @JJediny is the best person to support you in making sure the other DNS records are in a good place.

File: terraform/digitalgov.gov.tf [hunk 1]
```diff
@@ -5,6 +5,124 @@ resource "aws_route53_zone" "digitalgov_gov_zone" {
...
+}
+
+# usdigitalregistry.digitalgov.gov
+resource "aws_route53_record" "usdigitalregistry_digitalgov_gov_a" {
+  zone_id = "${aws_route53_zone.digitalgov_gov_zone.zone_id}"
+  name = "usdigitalregistry.digitalgov.gov."
+  type = "A"
+
+  alias {
+    name = "XXXXXXXXXXXXXXXXXXXX."
```
Comment (JJediny) [inline terraform/digitalgov.gov.tf]: @jeremyzilar how/when will these be completed?

File: terraform/digitalgov.gov.tf [hunk 2]
```diff
@@ -5,6 +5,124 @@ resource "aws_route53_zone" "digitalgov_gov_zone" {
...
+}
+
+# dap.digitalgov.gov
+resource "aws_route53_record" "dap_digitalgov_gov_a" {
+  zone_id = "${aws_route53_zone.digitalgov_gov_zone.zone_id}"
+  name = "dap.digitalgov.gov."
+  type = "A"
+
+  alias {
+    name = "XXXXXXXXXXXXXXXXXXXX."
```
Comment (JJediny) [inline terraform/digitalgov.gov.tf]: same as above

**Comment by wslack** [comment]
@jeremyzilar @JJediny shouldn't this be merged first and then the namservers redirected?

Also, John, did you check for any syntax issues or should I do that?

**Comment by jeremyzilar** [comment]
This one is not happening at this time.

**Pull request closed without merging**